clvm/operators.py

```diff
@@ -53,15 +62,13 @@ def f(args):
                 return self.unknown_op_handler(op, args)
         return f(arguments)
 
-    def set_unknown_op_handler(self, callback: Callable[[bytes, CLVMObject], Tuple[int, CLVMObject]]):
-        self.unknown_op_handler = callback
-
     def unknown_op_raise(self, op: bytes, arguments: CLVMObject):
         raise EvalError("unimplemented operator", arguments.to(op))
 
+QUOTE_ATOM = KEYWORD_TO_ATOM["q"]
+APPLY_ATOM = KEYWORD_TO_ATOM["a"]
```

Comment:
do we still need to "export" these from `clvm`? I would think we could just make this part of the default `OPERATOR_LOOKUP` exported.

Speaking of which, is there a way to make it immutable in python? To avoid the kind of bug I encountered last night?

Reply:
I agree, but I didn't want to make that change in the same PR.
I was looking into python immutability last night. We could probably get it by using a different base type, or removing some methods. Richard will know the best way - I'll pester him today!